releasenotes/notes/prepare-0.25-2efd7230b0ae0719.yaml

```diff
@@ -0,0 +1,98 @@
+---
+prelude: |
+  The Qiskit Terra 0.25.0 release highlights are:
+
+  * Control-flow operations are now supported through the transpiler at
```

Comment:
These are just copies of the release notes in the feature section right? Maybe we either want to have a more brief entry that just highlights the feature for the prelude, or if we're going to keep the full notes here we should remove the corresponding feature note.

Reply:
Yeah they're the same. IMO, it's important for these to be up top since it might help with adoption (at least in the case of the classical expression support), and I like the complete form for that. I don't mind these appearing twice, especially since subsequent entries from the same release note file will appear right after them in the corresponding sections below. I don't mind changing this though if you feel strongly.

Reply:
That's fine, lets go with this then. We haven't done this in the past, but lets give this a try and see how it works out.